node/core/backing/src/lib.rs

```diff
@@ -24,11 +24,11 @@ use std::sync::Arc;
 use bitvec::vec::BitVec;
 use futures::{
 	channel::{mpsc, oneshot},
-	task::{Spawn, SpawnError},
 	Future, FutureExt, SinkExt, StreamExt,
 };
 
 use keystore::KeyStorePtr;
+use sp_core::traits::SpawnNamed;
```

Comment:
To avoid every subsystem crate needing to depend on `sp_core`, maybe we can re-export this from `polkadot-node-primitives`.